crates/apub/src/protocol/activities/following/follow.rs

```diff
@@ -8,6 +12,8 @@ use url::Url;
 pub struct Follow {
   pub(crate) actor: ObjectId<ApubPerson>,
   /// Optional, for compatibility with platforms that always expect recipient field
+  #[serde(deserialize_with = "deserialize_opt_one")]
+  #[serde(default)]
```

Comment:
This can be written in a single line like 
  `#[serde(deserialize_with = "deserialize_skip_error", default)]`